src/smpdflib/core.py

```diff
@@ -719,6 +729,19 @@ def make_pdf_results(pdf, Q, flavors=None, xgrid=None):
     return results
 
 
+def make_pdfcorr_results(pdf, Q, flavors=None, xgrid=None):
```

Comment:
I think we need it because make_pdf_results returns results decomposed by flavors.